openstack/maia/templates/prometheus-deployment.yaml

```diff
@@ -41,6 +41,38 @@ spec:
               mountPath: /prometheus
             - name: config
               mountPath: /etc/prometheus
+{{- if .Values.prometheusv2.enabled }}
+        - name: prometheusv2
+          image: "{{ .Values.prometheusv2.image.repository }}:{{ .Values.prometheusv2.image.tag }}"
+          args:
+            - --config.file=/etc/prometheus/prometheusv2.yaml
```

Comment:
@auhlig i think we would have to provide this files then as well - would it have the same content as the v1 one?

Reply:
discussed with arno: same config file, so we will use prometheus.yaml